code/molecular_phenotypes/calling/RNA_calling.ipynb

```diff
@@ -433,6 +429,7 @@
     "cwd = path(f'{cwd:a}')\n",
     "# Reference gene model\n",
     "parameter: gtf = path\n",
+    "parameter: RSEM_index = path\n",
```

Comment:
this should not be a global parameter. It should be in RSEM step

Reply:
Is there a reason you must do this instead?

Reply:
It was a leftover from when I debugged, fixed it.